tutorials/fiori-ios-scpms-floorplan/fiori-ios-scpms-floorplan.md

```diff
@@ -431,7 +451,7 @@ You can now navigate back and forth between the `SupplierTableViewController` an
 
 [ACCORDION-BEGIN [Step 6: ](Implement the loading and displaying of supplier-specific products)]
 
-This view is similar to the `SupplierTableViewController` but instead of fetching all products, we will fetch supplier-specific products. To achieve that, we again can utilize the OData APIs. `SAPOData` provides the possibility to create so-called `DataQuery` objects which can define typical OData arguments for a backend call.
+This view is similar to the `SupplierTableViewController` but instead of fetching all products, we will fetch supplier-specific products. To achieve that, we again can utilise the OData APIs. `SAPOData` provides the possibility to create so-called `DataQuery` objects which can define typical OData arguments for a backend call.
```

Comment:
Please don't use the "royal we" unless you are referring to SAP. When giving instructions to the user, say "you" instead. Please also check the rest ot the tutorial and make necessary changes.